core/src/main/resources/error/error-classes.json

```diff
@@ -552,6 +552,12 @@
       "Unable to acquire <requestedBytes> bytes of memory, got <receivedBytes>"
     ]
   },
+  "UNPIVOT_REQUIRES_ATTRIBUTES" : {
+    "message" : [
+      "UNPIVOT requires given {given} to be Attributes when no {empty} are given: [<types>]"
```

Comment:
```suggestion
      "UNPIVOT requires given {given} to be columns when no {empty} are given, but got: [<expressions>]"
```

Reply:
then the caller side can just pass "id" and "value"

Reply:
Fixed in 2c8f53df, though `but got` implies `expressions` is an exhaustive list, but it is only the non-attributes.

I have rephrased that to

    ... expressions are given. These are not columns: [<expressions>].